apps/emqx/src/emqx_persistent_session_ds.hrl

```diff
@@ -64,10 +68,27 @@
     %%   message in the range.
     iterator :: emqx_ds:iterator(),
     %% Reserved for future use.
-    misc = #{} :: map()
+    misc = #{} :: #{
+        %% What commit tracks this range is part of.
+        %% This is rarely stored: we only need to persist it when the range
+        %% contains QoS 2 messages.
+        ?T_tracks => non_neg_integer(),
```

Comment:
Q: Are we already past the point where we can add new record fields here? This would slightly help to improve code readability.

Reply:
I don't think we enabled table creation on the master just yet. Should be fine to modify the schema?

Reply:
Agree, I also think it should. Will make the necessary changes.